code/modules/admin/topic.dm

```diff
@@ -3041,9 +3041,13 @@
 					if(answer1 == "Primary and Secondary")
 						secondary_effect = input(usr, "Which secondary effect would you like?", "Secondary effect") as null|anything in effects
 					var/obj/machinery/artifact/custom = new /obj/machinery/artifact(get_turf(usr))
```

Comment:
Making this spawn a subtype that doesn't initialize effects instantly would probably be better.